src/main/java/seedu/bankwithus/AccountList.java

```diff
@@ -29,4 +30,18 @@ public void addAccount(String name, String balanceString) throws NumberFormatExc
             ui.showNumberFormatError();
         }
     }
+
+    /**
+     * name and balance are separated by $ prepared to be saved
+     *
+     * @return returns all accounts details in String
+     */
+    public String getAllAccountDetails() {
+        String temp = "";
+        for (Account acc : accounts) {
+            temp += acc.name + "$" + acc.balance;
+            temp += "\n";
+        }
+        return temp;
+    }
```

Comment:
The way this is coded makes me feel a little uncomfortable. I would rather get a single account's details one by one in a loop in the parent function. I feel like something might go wrong if there are too many accounts to save in the save file. No need to change this, since it still works, but may affect future versions